src/Umbraco.Core/Persistence/Repositories/MediaRepository.cs

```diff
@@ -453,8 +453,14 @@ protected override int RecycleBinId
             Func<Tuple<string, object[]>> filterCallback = null;
             if (filter.IsNullOrWhiteSpace() == false)
             {
-                sbWhere.Append("AND (umbracoNode." + SqlSyntax.GetQuotedColumnName("text") + " LIKE @" + args.Count + ")");
+                sbWhere
+                    .Append("AND (umbracoNode.")

```

Comment:
Why not also make it extra safe and use `SqlSyntax.GetQuotedTableName("umbracoNode")` ?